### ISSUE TITLE
Properties in scm url are no longer interpolated?

### ISSUE THREAD

**Issue by garyhodgson** [danielflower/multi-module-maven-release-plugin#37]
Hi

I believe there is a regression introduced since version 2.0.4.  

We often use a property in the pom.xml scm tags, e.g. 

```
<scm>
<developerConnection>scm:git:ssh://git@git.xxxxx.com/abc/${project.artifactId}.git</developerConnection>
</scm>
```

In 2.0.3 this worked fine, but since 2.0.4 the following exception is thrown (e.g. during releaser:next)

> org.eclipse.jgit.api.errors.TransportException: ssh://git@git.xxxxx.com/abc/${project.artifactId}.git: GitLab: The project you were looking for could not be found.

It sounds as though the "${project.artifactId}" is now being sent directly to git, rather than the interpolated value.

**Comment by danielflower** [comment]
Ergh, a breaking change. Sorry. The intention behind the change was that if there is no scm tag at all, then it should not inherit it from a parent pom (as that pom would be in another git repo, so it would be wrong to use).

However, the change stopped the interpolation working. Working on a fix.

**Comment by danielflower** [comment]
Should be fixed in `2.0.8`

**Comment by garyhodgson** [comment]
Can confirm that 2.0.8 fixes this.  Thanks!
